molecule/core.py

```diff
@@ -61,6 +61,10 @@ def main(self):
         # merge in molecule.yml
         self._config.merge_molecule_file()
 
+        # ensure the .molecule directory exists
+        if not os.path.isdir(os.path.join(os.curdir, ".molecule")):
```

Comment:
This should leverage the config value molecule_dir: https://github.com/metacloud/molecule/blob/master/molecule/conf/defaults.yml#L19